integration/experiment/ffnet/README.md

```diff
@@ -0,0 +1,243 @@
+                       By default, it uses the sticker for the compute
+                       nodes, and executes an additional run with max
+                       turbo frequency as the limit.  Within each
+                       trial, the experiment script will start at the
+                       highest frequency and run at each limit until
+                       it reaches the minimum.
+
+  - `--min-frequency`: the minimum CPU frequency setting for the sweep.
+
+  - `--step-frequency`: the step size in hertz between CPU frequency
+                        settings for the sweep.
+ 
+  - `--run-max-turbo`: executes additional runs with max turbo
+                       frequency as the limit for the core frequency.
+
+  **CPU Uncore Frequency Settings**
+ 
+  - `--max-uncore-frequency`: the maximum uncore frequency setting for
+                              the sweep.  By default, it uses 2.7 GHz.
+
+  - `--min-uncore-frequency`: the minimum uncore frequency setting for
+                              the sweep.  By default, it uses 1.2 GHz.
+
+  - `--step-uncore-frequency`: the step size in hertz between uncore
+                               frequency settings for the sweep.
+
+  **GPU Frequency Settings**
+
+  - `--max-gpu-frequency`: the maximum GPU frequency setting for the sweep.
+                           By default, it uses the maximum frequency for the
+                           GPUs on the node. 
+
+  - `--min-gpu-frequency`: the minimum GPU frequency setting for the sweep.
+                           By default, it uses the minimum frequency for the
+                           GPUs on the node.
+
+  - `--step-gpu-frequency`: the step size in hertz between settings for the sweep.
+                            By default, it uses the minimum supported step size
+                            of GPUs on the node.
+
+  Note that for any frequency domain, if the values above are not specified or if
+  the min and max values are equal, a frequency sweep will not be executed across
+  that domain.
+
+## Supporting Files
+
+#### `gen_hdf_from_fsweep.py`:
+
+  Generates HDF files used to generate the neural net and frequency recommendation json files.
+  This takes report and trace files from frequency sweep experiments as inputs, checks for
+  required signals on CPU and GPU domains, annotates data with microbenchmark and node names,
+  and outputs the HDFs. Two files are generated: The stats file contains per-region report
+  information used to determine frequency recommendation for each region class. The trace file
+  contains trace data that is used to create a neural net that determines region class 
+  probabilities during a workload execution.
+
+  This script requires the following positional inputs:
+
+  - `output`: Prefix for output files `[output]_stats.h5` and `[output]_traces.h5`
+  - `frequency_sweep_dirs`: Directories containing reports and traces from frequency sweeps
+    
+  E.g.:
```

Comment:
Minor recommendation for consistency with other docs (applies to other instances in this file):

```suggestion
  Example:
```